extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -49,6 +53,7 @@ export class WizardController {
 			const canEnter = await pages[lastPage]?.canEnter() ?? true;
 
 			return canEnter && canLeave;
+			// return true;
```

Comment:
dead code

Reply:
Yeah when I test I just make that return true since I don't want to do any page validation. I'll keep switching this for the next few PRs and then go and clean up.